src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs

```diff
@@ -117,13 +107,13 @@ public void DesignTimeBuild ([Values(false, true)] bool isRelease, [Values (fals
 					if (useManagedParser) {
 						FileAssert.Exists (designTimeDesigner, $"{designTimeDesigner} should not have been deleted.");
 					}
-					items.Clear ();
+					var items = new List<string> ();
 					if (!useManagedParser) {
-						foreach (var file in Directory.EnumerateFiles (Path.Combine (intermediateOutputPath, "android"), "R.java", SearchOption.AllDirectories)) {
+						foreach (var file in Directory.EnumerateFiles (Path.Combine (intermediateOutputPath, "android", "src"), "R.java", SearchOption.AllDirectories)) {
```

Comment:
@garuma we discovered we had some duplicate copies of `R.java` in:
- `obj/Debug/android/com/mypackage/R.java`
- `obj/Debug/android/src/com/mypackage/R.java`

This PR is removing the non-`src` one. I wanted to check and make sure the designer doesn't use it?

Reply:
@jonathanpeppers that should be fine, we only use the compiled version and the resulting Resource.designer.cs

Thanks for checking!